Translations/translations_definitions.json

```
@@ -165,6 +165,12 @@
       "len": 1,
       "default": "F",
       "description": "Shown when the locking mode is set to lock all buttons."
+    },
+    {
+      "id": "SettingLockTempChar",
+      "len": 1,
+      "default": "T",
+      "description": "Shown when the locking mode is set to lock only the boost button's temperature change function"
```

In my humble opinion, the description is a bit of confusing. As far as I understand from the brief review of [the original bug report](#1663) and this pull-request, basically it should sound something like: _Shown when the locking mode is set to lock only A/+ button_.

Please, correct me if I'm wrong.

---

I worded it like this because the nature of the mode is much more different than the other locking modes.
It's not entirely locking the A/+ button, only part of it's functionality, 
```suggestion
      "description": "Shown when the locking mode is set to disable the A/+ button's temp change function"
```
Would this be a better description?